bundles/PersonalizationBundle/tests/Unit/Targeting/Document/TargetGroupEditableTest.php

```diff
@@ -51,14 +51,14 @@ public function testTargetGroupsEditable(): void
         $this->createTargetGroup($targetGroup2);
 
         $targetGroup1 = TargetGroup::getByName($targetGroup1);
-        $targetGroup1EditableData = $this->seed+1;
-        $targetGroupEditableName1 = $this->saveTargetGroupEditable($targetGroup1, $defaultEditableName, $targetGroup1EditableData);
+        $targetGroup1EditableData = $this->seed + 1;
+        $targetGroupEditableName1 = $this->saveTargetGroupEditable($targetGroup1, $defaultEditableName, (string) $targetGroup1EditableData);
 
         $this->testDataHelper->assertInput($this->testPage, $targetGroupEditableName1, $targetGroup1EditableData);
 
         $targetGroup2 = TargetGroup::getByName($targetGroup2);
-        $targetGroup2EditableData = $this->seed+ 2;
-        $targetGroupEditableName2 = $this->saveTargetGroupEditable($targetGroup2, $defaultEditableName, $targetGroup2EditableData);
+        $targetGroup2EditableData = $this->seed + 2;
+        $targetGroupEditableName2 = $this->saveTargetGroupEditable($targetGroup2, $defaultEditableName, (string) $targetGroup2EditableData);
```

Comment:
why do you prefer to cast the variable instead of leaving the methods parameter type at `int`?

Reply:
Is concated in the first line. So its a string. In 10.6 it is also defined as string. But we can change it back if you prefer your solution.

Reply:
but `$seed` is defined as `int`, or am I wrong? and an addition should not change that - right?

 https://github.com/pimcore/pimcore/pull/14517/files#diff-77eaa265c0cb4cbaf1a5cc4361450c6229ba0a224232e81386e591876ae35b52R26

Reply:
I mean the concating in line 85

Reply:
ah kk i see - yea then `string` is good I would say - thanks

Reply:
@Corepex I changed it in https://github.com/pimcore/pimcore/pull/14517/commits/f2c97e20ccc274703b3f418b14d34cd48d0fc9fc but change the name to seed to make it clear.
The assertInput method also have a seed param as integer

Reply:
LGTM - thanks